journal/original/03-chapter-2.adoc

```diff
@@ -579,21 +579,21 @@ and make them zealous for it in their generation; and good patterns to their off
 and those that might be seeking the Lord, who were yet strangers to him.
 
 After we had cleared ourselves in those parts,
-we got a shallop and went over Chesapeak bay to Accomack, not without some difficulty.
+we got a shallop and went over Chesapeak Bay to Accomack, not without some difficulty.
 The Lord`'s love was great to us, and his power manifested for our deliverance;
 blessed be his holy name for ever.
 We visited Friends on the eastern shore, and had good service.
 At one place, there came a great many Indians to a Friend`'s house where I was;
 one of them could speak very good English.
 I had some discourse with him about God, Christ, and religion: I asked him,
 if he believed there was a God?
-He said.
-Yes, and that all the Indians believed so too; but he was a Protestant,
-and lived amongst the English.
+He said,
+"`Yes, and that all the Indians believed so too; but he was a Protestant,
+and lived amongst the English.`"
```

Comment:
I think the way you did this passage in the modernized was preferable.  The quotation marks here don't feel exactly right, because the "he was a Protestant" would be "I am a Protestant" if it was actually a true quotation.  So, my thought would be either remove the quotes and do nothing, or switch to italics. Either one seems fine to me.